recipes/simbody/build.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# FIXME: This is a hack to make sure the environment is activated.
+# The reason this is required is due to the conda-build issue
+# mentioned below.
+#
+# https://github.com/conda/conda-build/issues/910
+source activate "${CONDA_DEFAULT_ENV}"
+
+mkdir build
+cd build
+if [[ "$OSTYPE" == "linux-gnu" ]]; then
+	cmake .. \
+		-LAH \
+		-DCMAKE_INSTALL_PREFIX="$PREFIX" \
+		-DBUILD_USING_OTHER_LAPACK="$PREFIX/lib/libopenblas.so" \
```

Comment:
To simplify extension handling, I normally define `DYLIB_EXT=dylib` and `DYLIB_EXT=so` using an `if` statement like this. Once we drop `-j$(nproc)` we should be able to move these build steps out of the `if`.